tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartLocaleGrowth.java

```diff
@@ -0,0 +1,347 @@
+            long found = 0;
+            long total = 0;
+            for (Entry<String, FoundAndTotal> entry : currentData.entrySet()) {
+                found += entry.getValue().found;
+                total += entry.getValue().total;
+            }
+            System.out.println("year\t" + year
+                + "\tversion\t" + version
+                + "\tlocales\t" + currentData.size()
+                + "\tfound\t" + found
+                + "\ttotal\t" + total
+                + "\tdetails\t" + currentData
+                );
+            out.flush();
+            if (latestData == null) {
+                latestData = currentData;
+            }
+            Counter2<String> completionData = getCompletion(latestData, currentData);
+            addCompletionList(year+"", completionData, growthData);
+            if (DEBUG) System.out.println(currentData);
+        }
+        boolean first = true;
+        for (Entry<String, List<Double>> entry : growthData.entrySet()) {
+            if (first) {
+                for (int i = 0; i < entry.getValue().size(); ++i) {
+                    out.print("\t" + i);
+                }
+                out.println();
+                first = false;
+            }
+            out.println(entry.getKey() + "\t" + Joiner.on("\t").join(entry.getValue()));
+        }
+    }
+
+    private static final class ReleaseInfo {
+        private ReleaseInfo(VersionInfo versionInfo, int year) {
+            this.version = versionInfo;
+            this.year = year;
+        }
+        VersionInfo version;
+        int year;
+    }
+
+    // TODO merge this into ToolConstants, and have the version expressed as VersionInfo.
+    private static final List<ReleaseInfo> versionToYear;
+    static {
+        Object[][] mapping = {
+            { VersionInfo.getInstance(43), 2023 },
+            { VersionInfo.getInstance(42), 2022 },
+            { VersionInfo.getInstance(40), 2021 },
+            { VersionInfo.getInstance(38), 2020 },
+            { VersionInfo.getInstance(36), 2019 },
+            { VersionInfo.getInstance(34), 2018 },
+            { VersionInfo.getInstance(32), 2017 },
+            { VersionInfo.getInstance(30), 2016 },
+            { VersionInfo.getInstance(28), 2015 },
+            { VersionInfo.getInstance(26), 2014 },
+            { VersionInfo.getInstance(24), 2013 },
+            { VersionInfo.getInstance(22,1), 2012 },
+            { VersionInfo.getInstance(2,0,1), 2011 },
+            { VersionInfo.getInstance(1,9,1), 2010 },
+            { VersionInfo.getInstance(1,7,2), 2009 },
+            { VersionInfo.getInstance(1,6,1), 2008 },
+            { VersionInfo.getInstance(1,5,1), 2007 },
+            { VersionInfo.getInstance(1,4,1), 2006 },
+            { VersionInfo.getInstance(1,3), 2005 },
+            { VersionInfo.getInstance(1,2), 2004 },
+            { VersionInfo.getInstance(1,1,1), 2003 },
+        };
```

Comment:
a third (fourth?) list of versions?

Reply:
can you put a link to my unscheduled ticket https://unicode-org.atlassian.net/browse/CLDR-9149 (maybe we should schedlue it)